docs/products/kafka/kafka-connect/howto/jdbc-source-connector-mysql.rst

```diff
@@ -0,0 +1,121 @@
+* ``MYSQL_PASSWORD``: The database password for the ``MYSQL_USER``
+* ``MYSQL_DATABASE_NAME``: The database name
+* ``MYSQL_TABLES``: The list of database tables to be included in Apache Kafka; the list must be in the form of ``table_name1,table_name2``
+
+.. Note::
+
+    If you're using Aiven for MySQL the above details are available in the `Aiven console <https://console.aiven.io/>`_ service Overview tab or via the dedicated ``avn service get`` command with the :ref:`Aiven CLI <avn_service_get>`.
+
+Setup a MySQL JDBC source connector with Aiven CLI
+-------------------------------------------------------
+
+The following example demonstrates how to setup an Apache Kafka JDBC source connector to a MySQL database using the :ref:`Aiven CLI dedicated command <avn_service_connector_create>`.
+
+Define a Kafka Connect configuration file
+'''''''''''''''''''''''''''''''''''''''''
+
+Define the connector configurations in a file (we'll refer to it with the name ``jdbc_source_mysql.json``) with the following content:
+
+::
+
+    {
+        "name":"CONNECTOR_NAME",
+        "connector.class":"io.aiven.connect.jdbc.JdbcSourceConnector",
+        "connection.url":"jdbc:mysql://MYSQL_HOST:MYSQL_PORT/MYSQL_DATABASE_NAME?&verifyServerCertificate=false&useSSL=true&requireSSL=true",
+        "connection.user":"MYSQL_USER",
+        "connection.password":"MYSQL_PASSWORD",
+        "table.whitelist":"MYSQL_TABLES",
+        "mode":"JDBC_MODE",
+        "topic.prefix":"KAFKA_TOPIC_PREFIX",
+        "tasks.max":"NR_TASKS",
+        "poll.interval.ms":"POLL_INTERVAL"
+    }
+
+The configuration file contains the following entries:
+
+* ``name``: the connector name
+* ``MYSQL_HOST``, ``MYSQL_PORT``, ``MYSQL_DATABASE_NAME``, ``MYSQL_USER``, ``MYSQL_PASSWORD`` and ``MYSQL_TABLES``: source database parameters collected in the :ref:`prerequisite <connect_jdbc_mysql_source_prereq>` phase. 
+* ``mode``: the query mode, more information in the :doc:`dedicated page <../concepts/jdbc-source-modes>`; depending on the selected mode, additional configuration entries might be required.
+* ``topic.prefix``: the prefix that will be used for topic names. The resulting topic name will be the concatenation of the ``topic.prefix`` and the schema and table name.
+* ``tasks.max``: maximum number of tasks to execute in parallel. By default is 1, the connector can use at max 1 task for each source table defined.
+* ``poll.interval.ms``: query frequency, default 5000 milliseconds
+
+Check out the `dedicated documentation <https://github.com/aiven/jdbc-connector-for-apache-kafka/blob/master/docs/source-connector-config-options.rst>`_ for the full list of parameters.
+
+Create a Kafka Connect connector with Aiven CLI
+'''''''''''''''''''''''''''''''''''''''''''''''
+
+To create the connector, execute the following :ref:`Aiven CLI command <avn_service_connector_create>`, replacing the ``SERVICE_NAME`` with the name of the Aiven service where the connector needs to run:
```

Comment:
where the connector needs to run:

is not the user who has to run this?

Reply:
The connector runs in Aiven, in the Kafka cluster or in a dedicated Kafka connect cluster